docs/installation/server-config/config-reference.rst

```diff
@@ -198,14 +198,10 @@ When deploying KSQL to production, the following settings are recommended in you
     # will not result in data loss.
     ksql.streams.producer.retries=2147483647
 
-    # Set the batch expiry to Long.MAX_VALUE to ensure that queries will not
+    # Set the batch expiry to Integer.MAX_VALUE to ensure that queries will not
     # terminate if the underlying Kafka cluster is unavailable for a period of
     # time.
-    ksql.streams.producer.confluent.batch.expiry.ms=9223372036854775807
-
-    # Allows more frequent retries of requests when there are failures,
-    # enabling quicker recovery.
-    ksql.streams.producer.request.timeout.ms=300000
```

Comment:
@xvrl Just to check, I thought C3 was at 300s?

Reply:
This was changed in https://github.com/confluentinc/blueway/commit/33576c83ef515d19630dbf9af0085df4feab2039